lib/rex/exceptions.rb

```diff
@@ -156,15 +157,21 @@ def initialize(addr = nil, port = nil)
   #
   def addr_to_s
     if host and port
-      "(#{host}:#{port})"
+      host.include?(':') ? "([#{host}]:#{port})" : "(#{host}:#{port})"
```

Comment:
If there's a colon in the host, it's probably an IPv6 address so wrap it in brackets before appending another colon and the port so the user can tell if there's a port specified at all. Unfortunately `Rex::Socket.is_ipv6?` can't be used because `rex-socket` is not a dependency of `rex-core`.